waspc/data/Cli/templates/new/src/client/jsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is there some stuff here that we don't want the users to touch?
Should we document it somehow, if so? Maybe directly in this file? I know JSON by itself doesn't support comments but there are tricks to add them.

Reply:
Yes, definitely, I'll put it in.